Doc/library/inspect.rst

```diff
@@ -40,11 +40,28 @@ attributes:
 +-----------+-------------------+---------------------------+
 | Type      | Attribute         | Description               |
 +===========+===================+===========================+
-| module    | __doc__           | documentation string      |
+| module    | __name_           | name of the module        |
++-----------+-------------------+---------------------------+
+|           | __doc__           | documentation string      |
++-----------+-------------------+---------------------------+
+|           | __cached__        | the path where a compiled |
+|           |                   | version is/should be      |
+|           |                   | stored                    |
++-----------+-------------------+---------------------------+
+|           | __path__          | list of strings specifying|
+|           |                   | the search path within a  |
+|           |                   | package (not set for      |
+|           |                   | modules)                  |
++-----------+-------------------+---------------------------+
+|           | __package__       | Parent package for the    |
+|           |                   | module/package            |
```

Comment:
Can you lowercase the description in the __package__ row and __loader__ row so that capitalization is consistent?